src/model.d.ts

```diff
@@ -570,6 +570,38 @@ export interface WhereOperators<AttributeType = any> {
    * https://www.postgresql.org/docs/14/functions-range.html
    */
   [Op.adjacent]?: WhereOperators<AttributeType>[typeof Op.strictLeft];
+
+  /**
+   * PG only
+   *
+   * Check if any of these array strings exist as top-level keys.
+   *
+   * @example
+   * ```typescript
+   * { jsonbAttribute: { [Op.anyKeyExists]: ['a','b'] } }
+   * // results in
+   * // "jsonbAttribute" ?| ARRAY['a','b']
+   * ```
+   *
+   * https://www.postgresql.org/docs/9.5/functions-json.html
+   */
+  [Op.anyKeyExists]?: WhereOperators<StaticValues<NonNullable<string>>> | Literal;
```

Comment:
`WhereOperators` refers to this type. The reason some types use `WhereOperators` is to re-use the same type as another operator.

The following makes `iRegexp` use the same type as `regexp`

```ts
[Op.iRegexp]?: WhereOperators<AttributeType>[typeof Op.regexp];
```

And `regexp` has the proper type definition:

```ts
[Op.regexp]?: AllowAnyAll<OperatorValues<Extract<AttributeType, string>>>;
```

When crafting the type you need to think about the following:

- are dynamic values allowed here (e.g. column references, functions, etc)
- is Op.any & Op.all usable in combination with this?